### PR TITLE
finallywordle

### PR DESCRIPTION
**Pull request by SanidhyaSharma28**
can you check if its fine?

### CODE REVIEW

File: src/styles/pages/games/Wordle.css [hunk 1]
```diff
@@ -0,0 +1,134 @@
+.Wordle {
+    text-align: center;
+    background-color: black;
+    width: 100vw;
+    height: 100vh;
```
Comment (Ishwarendra) [inline src/styles/pages/games/Wordle.css:5]: Don't hardcode these values. 

image

There should not be any horizontal scrollbars.

File: src/styles/pages/games/Wordle.css [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    /* display: flex; */
+    color: white;
+  }
+  
+  body {
```
Comment (Ishwarendra) [inline src/styles/pages/games/Wordle.css:10]: dont' add css to `body`, `h1`, `div` `p` and similar tags that can affect other components as well.

File: src/styles/pages/games/Wordle.css [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    align-items: center;
+    padding-top: 5px;
+    flex-direction: column;
+  }
+
+  .wordleWhite{
+    color: white;
+  }
+  
+  .board {
```
Comment (Ishwarendra) [inline src/styles/pages/games/Wordle.css:43]: Name css class like `wordleBoard` instead of `board`.

**Review by Ishwarendra — CHANGES_REQUESTED** [review]
Also complete black doesn;t look too good. Try to keep light background for now to match other games and activities.

**Comment by SanidhyaSharma28** [comment]
@Ishwarendra can you check now???

**Review by Ishwarendra — APPROVED** [review]
Good work.

**Pull request merged**